controllers/google.auth.js

```diff
@@ -0,0 +1,44 @@
+const jwt = require("jsonwebtoken");
+
+exports.loginFailed = (req, res) => {
+    res.status(401).json({
+        message: 'Failed Login'
+    })
+}
+
+/**
+ * @author Timothy Adeyeye <adeyeyetimothy33@gmail.com>
+ * @description Authenticaton with Google OAuth
+ * @route `/api/v1/auth/register`
+ * @access Public
+ * @type GET
+ */
+exports.profile = (req, res) => {
+    const user = req.user
```

Comment:
Did Passport Write `req.user` this to memory

Reply:
![serialize](https://user-images.githubusercontent.com/49001006/206638113-14607c88-59d8-4b61-8c06-946a949df4b2.png)

Yes, I think from the serialize method